src/screens/LoginScreen.tsx

```diff
@@ -37,6 +38,25 @@ export default function Login({ navigation }) {
   });
   const attemptInvisibleVerification = false;
 
+  useEffect(() => {
+    if (verificationId) {
+      showMessage({
+        text: 'Verification code has been sent to your phone.',
+      });
+      navigation.navigate('Verify', {
+        verificationId,
+      });
+    }
+  }, [verificationId, navigation]);
+
+  function phoneFormatter(num: string) {
+    if (num.length === 10) {
+      setPhoneNumber('+1'.concat(num));
```

Comment:
I was thinking more of a regex instead of a concat, feel free to use some third party

https://www.npmjs.com/package/react-phone-number-input

(use yarn though)

Reply:
phone formatting get's weird, also it needs to verify its correct, etc.

Reply:
@nikkisuzani you don't need phoneFormatter anymore, `PhoneInput` should do that for you automatically

can you also check if it's a valid phone number? there's utility functions you can import in this library

Reply:
"I personally don't use isValidPhoneNumber() for phone number validation in my projects. The rationale is that telephone numbering plans can and sometimes do change, meaning that isValidPhoneNumber()function may one day become outdated on a website that isn't actively maintained anymore. Imagine a "promo-site" or a "personal website" being deployed once and then running for years without any maintenance, where a client may be unable to submit a simple "Contact Us" form just because this newly allocated pool of mobile phone numbers wasn't present in that old version of libphonenumber-js bundled in it."

^^ This is from the documentation -- so I'm using isPossiblePhoneNumber() instead of isValidPhoneNumber() but if you want me to change it I can.